osbs/build/build_request.py

```diff
@@ -791,6 +832,9 @@ def render(self, validate=True):
         self.render_koji_promote(use_auth=use_auth)
         self.render_sendmail()
 
+
+        self.render_customizations()
```

Comment:
I wasn't sure if it should be done at the beginning or at the end because if someone sets args for a site customizations, won't some of these other `render_` function calls potentially overwrite those customizations? I could split it out to be `self.render_customizations_remove()` and `self.render_customizations_enable()` with one at the beginning and the other at the end.